public/agileui.css

```diff
@@ -86,6 +90,16 @@ i.atk-panel-warning {
 i.atk-panel-warning.atk-visible {
   color: #F2711C;
 }
+.atk-admin-left-menu {
+  top: 47px !important;
```

Comment:
Can the rule be rewritten without `!important` - it is otherwise hard to override this rule in custom css.

Reply:
It is important because, without it, it will not work. 😄 Besides, it goes with the specific template.

Reply:
I meant more specific selector, or does any other rule already use `!important`?